content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -10,6 +10,9 @@ layout: bundle
 
 #### Implemented
 
+##### Breaking change in REST APIs
+
+As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of release 10.18+, wrong values for `pageSize` or `currentPage` result in a response code 422 instead of 500.
```

Comment:
```suggestion
As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of release 10.18+, wrong values for `pageSize` or `currentPage` result in a 422 (Unprocessable entity) HTTP status code instead of a 500 (Internal server error) HTTP status code.
```